src/item_db.py

```diff
@@ -3,7 +3,7 @@
 
 IMAGE_ROOT_URL = "https://static.wikia.nocookie.net/terraria_gamepedia/images/"
 WIKI_ROOT_URL = "https://terraria.fandom.com/wiki/"
-ITEMS_JSON_PATH = "data/items.json"
+ITEMS_JSON_PATH = r"E:\python\terraria\terraria-journey-tracker-server\data\items.json"
```

Comment:
Hmmmmm. This is something we may want to use a relative path using the `os.path` stuff. 
Basically
- get the dir of the current execution
- append the elements

Check
- https://docs.python.org/3/library/os.path.html
- https://docs.python.org/3/library/os.path.html#os.path.join

To get the path for the current working directory (which is where the script was invoked, which we can assume to be the same as the folder with `start.py`)
```py
import os 
os.path.abspath(os.getcwd())
```